test/uk/gov/hmrc/helptosave/repo/EligibilityStatsStoreSpec.scala

```diff
@@ -32,32 +40,52 @@ class EligibilityStatsStoreSpec extends TestSupport with MongoSupport {
     "aggregating the eligibility stats" must {
 
       "return results as expected" in {
-        val store = newEligibilityStatsMongoStore(reactiveMongoComponent)
 
-        await(store.collection.insert(ordered = false).one(document))
-        await(store.getEligibilityStats) shouldBe List(EligibilityStats(Some(7), Some("Digital"), 1))
+        await(repository.collection.insertOne(EnrolmentData(nino              = randomNINO(), itmpHtSFlag = false, eligibilityReason = Some(7), source = Some("Digital"))).toFuture())
+        await(repository.getEligibilityStats) shouldBe List(EligibilityStats(Some(7), Some("Digital"), 1))
       }
     }
 
     "handle error while reading from mongo" in {
-      val store = newEligibilityStatsMongoStore(reactiveMongoComponent)
 
-      await(store.getEligibilityStats) shouldBe List.empty
+      await(repository.getEligibilityStats) shouldBe List.empty
     }
 
     "return aggregated results when there is more than one result" in {
       val document2 = Json.obj("eligibilityReason" -> 7, "source" -> "Digital", "total" -> 1).value
       val document3 = Json.obj("eligibilityReason" -> 8, "source" -> "Digital", "total" -> 1).value
-      val store = newEligibilityStatsMongoStore(reactiveMongoComponent)
 
-      await(store.collection.insert(ordered = false).one(document))
-      await(store.collection.insert(ordered = false).one(document2))
-      await(store.collection.insert(ordered = false).one(document3))
+      await(repository.collection.insertOne(
+        EnrolmentData(
```

Comment:
This object can be stored in val and reused when you did second insert at line 67, as both values are same

Reply:
ok